src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -73,17 +83,44 @@
 		public static Subsystem[] mainSubsystems;
 		public static CustomXbox driverXbox;
 		public static CustomJoystick operatorStick;
+		public static NavX navx;
+		public static MotionController chassisMC;
+		public static MotionController chassisEncoderMC;
 		// Vision
 		public static AligningCamera alignCamera;
 	}
 	
+	public static class HumanInput {
+		public static class Driver {
+			public static CustomXbox xbox;
+		}
+		
+		public static class Operator {
+			public static CustomJoystick stick;
+		}
+	}
+	
 	public RobotMap() {
 		Component.pdp = new PDP();
+		// NavX
+		Component.navx = new NavX(SerialPort.Port.kMXP);
+		// Component.leftWheel.setInverted(true);
+		Component.rightWheelEncoder = new CANEncoder(Port.CAN.rightEncoder);
+		Component.rightWheel = new PositionEncodedMotor("rightWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.rightWheelEncoder), new VictorSP(Port.PWM.rightDriveMotor));
+		Component.rightWheel.disablePID(); // TODO add encoders
 		// Chassis
 		Component.leftWheelEncoder = new CANEncoder(Port.CAN.leftEncoder);
 		Component.leftWheelEncoder.setReverseDirection(true);
```

Comment:
Agreed, although I would be OK with this pull request being expanded to include other autonomous driving capabilities.